From 94d8f4e56e5b98e907f6a80a318a7c5608343637 Mon Sep 17 00:00:00 2001
From: Maruthi Srinivas Bayyavarapu <maruthi.srinivas.bayyavarapu@xilinx.com>
Date: Fri, 30 Nov 2018 16:17:25 +0530
Subject: [PATCH 0131/1852] dt-bindings: ASoC: xlnx,spdif: Document spdif
 bindings

commit d1c93ef8f3560b6cb12d909e10661f9db61c6562 from
https://github.com/Xilinx/linux-xlnx.git

Added documentation for SPDIF IP core DT bindings.

Signed-off-by: Maruthi Srinivas Bayyavarapu <maruthi.srinivas.bayyavarapu@xilinx.com>
Reviewed-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/sound/xlnx,spdif.txt  | 21 +++++++++----------
 1 file changed, 10 insertions(+), 11 deletions(-)

diff --git a/Documentation/devicetree/bindings/sound/xlnx,spdif.txt b/Documentation/devicetree/bindings/sound/xlnx,spdif.txt
index 15c2d64d247c..600ab5cbefcd 100644
--- a/Documentation/devicetree/bindings/sound/xlnx,spdif.txt
+++ b/Documentation/devicetree/bindings/sound/xlnx,spdif.txt
@@ -1,28 +1,27 @@
-Device-Tree bindings for Xilinx SPDIF IP
+Device-Tree bindings for Xilinx SPDIF PL IP
 
-The IP supports playback and capture of SPDIF audio
+The IP supports SPDIF based playback and capture audio
 
 Required properties:
  - compatible: "xlnx,spdif-2.0"
- - clock-names: List of input clocks.
-   Required elements: "s_axi_aclk", "aud_clk_i"
- - clocks: Input clock specifier. Refer to common clock bindings.
+ - clocks: List of phandles to AXI4-Lite clock,
+	   Audio clock input used at the SPDIF/AES3 interface
+ - clock-names: Must contain "s_axi_aclk", "aud_clk_i" in the same order as
+		clocks listed in clocks property
  - reg: Base address and address length of the IP core instance.
  - interrupts-parent: Phandle for interrupt controller.
  - interrupts: List of Interrupt numbers.
- - xlnx,spdif-mode: 0 :- receiver mode
-		    1 :- transmitter mode
- - xlnx,aud_clk_i: input audio clock value.
+ - xlnx,spdif-mode: 0 :- receiver mode ; 1 :- transmitter mode
+ - xlnx,snd-pcm: phandle to audio formatter node
 
 Example:
 	spdif_0: spdif@80010000 {
 		clock-names = "aud_clk_i", "s_axi_aclk";
-		clocks = <&misc_clk_0>, <&clk 71>;
+		clocks = <&si570_1>, <&clk 71>;
 		compatible = "xlnx,spdif-2.0";
-		interrupt-names = "spdif_interrupt";
 		interrupt-parent = <&gic>;
 		interrupts = <0 91 4>;
 		reg = <0x0 0x80010000 0x0 0x10000>;
 		xlnx,spdif-mode = <1>;
-		xlnx,aud_clk_i = <49152913>;
+		xlnx,snd-pcm = <&audio_formatter_0>;
 	};
-- 
2.31.1

